engine/common/host.c

```diff
@@ -862,7 +862,9 @@ void Host_FreeCommon( void )
 
 	Mem_FreePool( &host.mempool );
 }
-
+#ifdef PANDORA
+int noshouldermb = 0;
+#endif
 /*

```

Comment:
What is the meaning of noshouldermb? 
Maybe better define it as unchangeable cvar and add to main parser?